package/src/index.ts

```diff
@@ -70,6 +92,260 @@ export class CandleClient {
   }): Promise<string> {
     return this.candle.executeTool(tool);
   }
+
+  public async getAssetAccounts(
+    query: AssetAccountQuery = {
+      linkedAccountIDs: undefined,
+      assetKind: undefined,
+    }
+  ): Promise<AssetAccount[]> {
+    const accounts = await this.candle.getAssetAccounts(query);
+    return accounts.map(this.convertToAssetAccount);
+  }
+
+  public async getTrades(
+    query: {
+      gainedAssetKind?: TradeQueryAssetKind;
+      lostAssetKind?: TradeQueryAssetKind;
+      counterpartyKind?: "merchant" | "user" | "service";
+    } & TradeQuery = {
+      counterpartyKind: undefined,
+      gainedAssetKind: undefined,
+      lostAssetKind: undefined,
+      linkedAccountIDs: undefined,
+      dateTimeSpan: undefined,
+    }
+  ): Promise<
+    {
+      dateTime: string;
+      state: TradeState;
+      counterparty: Counterparty;
+      lost: TradeAsset;
+      gained: TradeAsset;
+    }[]
```

Comment:
#nth Define `interface`s or `types` for all request/response types so that they show up more cleanly in the VSC hover popup box

Reply:
Will in a follow up PR.